src/components/Shared/ReceiveModal/ReceiveModal.tsx

```diff
@@ -35,23 +35,33 @@ const ReceiveModal: FC<Props> = ({ onClose }) => {
   const [comment, setComment] = useState("");
   const [isLoading, setIsLoading] = useState(false);
   const [copyAddress, addressCopied] = useClipboard(address);
+  const [error, setError] = useState("");
 
   const lnInvoice = invoiceType === TxType.LIGHTNING;
 
   const changeInvoiceHandler = async (txType: TxType) => {
     setAddress("");
     setAmount(0);
     setComment("");
+    setError("");
 
     setInvoiceType(txType);
 
     if (txType === TxType.ONCHAIN) {
       setIsLoading(true);
-      const resp = await instance.post("lightning/new-address", {
-        type: "p2wkh",
-      });
-      setAddress(resp.data);
-      setIsLoading(false);
+      await instance
+        .post("lightning/new-address", {
+          type: "p2wkh",
+        })
+        .then((resp) => {
+          setAddress(resp.data);
+        })
+        .catch((err) => {
+          setError(`${t("login.error")}: ${err.response?.data?.detail}`);
```

Comment:
Receive modal is using `login.error`? Is this intended or just a same needed word?

Reply:
Same needed word. 
Translates to `An error occured`

Will move some translations to a more "common" parent, but currently just need to get finished :)

Reply:
Created a issue #268